packages/FSharp.Formatting/FSharp.Formatting.fsx

```diff
@@ -1,76 +0,0 @@
-#nowarn "211"
```

Comment:
Ahh, I suspect it was deleted for me and then not re-generated? I'll add it back. :-)